docs/pages/versions/unversioned/sdk/age-range.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: AgeRange
+description: A library that provides access to age range information using Google Play Age Signals API on Android and Declared Age Range framework on iOS.
```

Comment:
```suggestion
description: A library that provides access to age range information using Play Age Signals API on Android and Declared Age Range framework on iOS.
```

I don't the official documentation prefixing it with "Google": https://developer.android.com/google/play/age-signals/use-age-signals-api#:~:text=The%20Play%20Age%20Signals%20API%20only%20returns%20data%20for%20users,the%20applicable%20jurisdiction%20and%20regions.